DQM/HcalCommon/interface/Constants.h

```diff
@@ -32,17 +32,25 @@ namespace hcaldqm
 			24, 0, 20, 0, 21, 0, 25, 0, 31, 0,
 			35, 0, 37, 0, 34, 0, 30, 0, 22,22,
 			29,29, 32,32, 0, 0, 0, 0, 0, 0,
-			0, 0, 36, 0, 0, 0, 0, 0, 0, 0,
+			0, 0, 38, 0, 0, 0, 0, 0, 0, 0,
 			0, 0, 0, 0, 0, 0, 0, 0, 0, 0
 		};
 		uint16_t const CRATE2FED[50] = {
```

Comment:
Hi @vkhristenko @deguio- ok, so these are indeed not constant constants (as we had worried about last summer). So, are these only for online DQM? I guess the answer is no, as the tier0 sees problems. If not, how do we support 2016 data?